tiledb/ml/readers/pytorch.py

```diff
@@ -45,6 +48,8 @@ def __init__(
         :param batch_size: The size of the batch that the generator will return. Remember
             to set batch_size=None when calling the PyTorch Dataloader API, because
             batching is taking place inside the TileDB IterableDataset.
+        :param batch_shuffle: Bool. True if we want to shuffle batches.
+        :param within_batch_shuffle: Bool. True if we want to shuffle records in each batche.
```

Comment:
- Don't need to repeat the type in the docstring, it's annotated
- Typo in the second line (`batche`)